src/PayID/pay-id-utils.ts

```diff
@@ -49,3 +53,24 @@ export default class PayIDUtils {
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   private constructor() {}
 }
+
+/**
+ * A static utility class for PayID.
+ *
+ * @deprecated Please use the idiomatically named `PayIdUtils` class instead.
+ */
+export class PayIDUtils {
+  /**
+   * Parse a PayID string to a set of PayIDComponents object
+   *
+   * @param parsePayID - The input Pay ID.
+   * @returns A PayIDComponents object if the input was valid, otherwise undefined.
+   */
+  public static parsePayID(payID: string): PayIDComponents | undefined {
+    return PayIdUtils.parsePayID(payID)
+  }
+
+  /** Please do not instantiate this static utility class. */
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  private constructor() {}
```

Comment:
Good comment.

Total aside: There's a `@typescript-eslint` rule that disallows using "classes as namespaces", which a class with only static methods is. I know that's a pretty common pattern in `xpring-common-js`, and I just wanted to get a feel for how you felt about that.

[Here is the actual rule documentation for some reasoning and examples](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-extraneous-class.md).